Expensify/App
# Update Address search - show all inputs

Added reviewers from [previous PR](https://github.com/Expensify/App/pull/6200) because you already have some context and saw most of these changes :)

Similar to the previous PR, but without moving the error handling to the form's state. I did a work around to batch clearing errors and kept the errors in Onyx.

### Details

Allow the user to manually complete address information and all fields are visible:

- Added `state`, `city` and `zip code` fields back in the following steps: "Company Step", "Personal information (requestor)" and "Additional information (beneficial owners)"
- Added `state` and `zip code` to AddDebitCardPage
- When we select a result from the address autocomplete, we use the information that is available. If some part is missing (i.e. city), we just ignore that.
- Deleted unused validation method for Google places address components

### Fixed Issues

$ https://github.com/Expensify/App/issues/6309

### Tests / QA steps

1. Create NewDot account, verify it.
2. Create workspace and start the add VBA flow
3. In "Company information" step:
    1. You should see the inputs: `Company address`, `City`, `Zip Code`, and `State`:
        <img src="https://user-images.githubusercontent.com/87341702/141878056-fa146760-ddd5-4717-9346-d3a21ae836aa.png" width="368">
    2. Click "Save & Continue", they should get inline errors
        <img src="https://user-images.githubusercontent.com/87341702/141878184-5bead448-7690-4dde-ba60-b49df7084038.png" width="368">
    3. Typing in any of the field should clear the error.
    4. Type and address in `Company address` and select a result. The other address fields (`City`, `Zip Code`, and `State`) should get updated with the information from the selected result.
    5. Focus on the `Company address` field and type `#12334` at the end (adding the apt number).
    6. Click "Save & Continue", you should be able to proceed to the next step.
4. Do the same steps (sub-steps in step (3)) for "Personal information"
5. Do the same steps (sub-steps in step (3)) for "Additional information" adding beneficial owners
6. Navigate to `settings/payments/add-debit-card` and do the same steps (sub-steps in step (3)). The only difference in this view is that the `City` field is not available.

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
